cmd/sgx-device-plugin/main.go

```diff
@@ -12,7 +15,33 @@ import (
 	"github.com/AliyunContainerService/sgx-device-plugin/pkg/utils"
 )
 
+var help bool
+
+func usage() {
+	fmt.Fprintf(os.Stdout, `sgx-device-plugin
+Usage: sgx-device-plugin [-ah]
+
+Options:
+`)
+	flag.PrintDefaults()
+}
+
+func init() {
+	flag.BoolVar(&help, "help", false, "Help for sgx-device-plugin")
```

Comment:
This help flag is built in and activated after flag.Parse(), I think we should remove it and related codes.